### PR TITLE
adding direct test for GenotypeGvcfs from genomicsdb

### PR DESCRIPTION
**Pull request by lbergelson**
@droazen

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java [hunk 1]
```diff
@@ -68,7 +76,7 @@
                 {getTestFile("spanningDel.delOnly.g.vcf"), getTestFile( "spanningDel.delOnly.gatk3.7_30_ga4f720357.expected.vcf"), NO_EXTRA_ARGS, b37_reference_20_21},
                 {getTestFile("spanningDel.depr.delOnly.g.vcf"), getTestFile( "spanningDel.depr.delOnly.gatk3.7_30_ga4f720357.expected.vcf" ), NO_EXTRA_ARGS, b37_reference_20_21},
                 {getTestFile("ad-bug-input.vcf"), getTestFile( "ad-bug-gatk3.7_30_ga4f720357-output.vcf"), NO_EXTRA_ARGS, b37_reference_20_21}, //Bad AD Propagation Haploid Bug
-                {getTestFile("CEUTrio.20.21.gatk3.4.g.vcf"), getTestFile( "CEUTrio.20.21.gatk3.7_30_ga4f720357.expected.vcf"), Arrays.asList("--dbsnp", "src/test/resources/large/dbsnp_138.b37.20.21.vcf"), b37_reference_20_21},
+                {getTestFile(CEUTRIO_20_21_GATK3_4_G_VCF), getTestFile(CEUTRIO_20_21_EXPECTED_VCF), Arrays.asList("--dbsnp", "src/test/resources/large/dbsnp_138.b37.20.21.vcf"), b37_reference_20_21},
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: Replace `src/test/resources/large/` with inherited constant for the large dir

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java [hunk 2]
```diff
@@ -141,6 +149,34 @@ public void testGenotypesOnly(File input, File expected, List<String> extraArgs,
...
+        return new Object[][]{
+                {getTestFile(BASE_PAIR_GVCF), getTestFile(BASE_PAIR_EXPECTED), new SimpleInterval("20", 1, 11_000_000), b37_reference_20_21},
+                {getTestFile(CEUTRIO_20_21_GATK3_4_G_VCF), getTestFile("CEUTrio.20.gatk3.7_30_ga4f720357.expected.vcf"), new SimpleInterval("20", 1, 11_000_000), b37_reference_20_21},
+                {getTestFile(CEUTRIO_20_21_GATK3_4_G_VCF), getTestFile("CEUTrio.21.gatk3.7_30_ga4f720357.expected.vcf"), new SimpleInterval("21", 1, 11_000_000), b37_reference_20_21}
+        };
+    }
+
+
+    @Test(dataProvider = "getGVCFsForGenomicsDB")
+    public void assertMatchingGenotypesFromTileDB(File input, File expected, htsjdk.samtools.util.Locatable interval, String reference) throws IOException {
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: import `Locatable`?

**Comment by droazen** [comment]
:+1: looks fine to me -- merge after resolving conflict

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java (hunk 1 above)
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: done

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java (hunk 2 above)
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java]: fixed

**Pull request merged**